src/collective/cover/static/compose.js

```diff
@@ -1,7 +1,7 @@
 
 (function ($) {
     $.fn.liveSortable = function (opts) {
-        this.live("mouseover", function() {
+        $(document).on("mouseover", this, function() {
```

Comment:
@cguardia compatibility with jQuery 1.9? I think this is already fixed on master